pageserver/src/tenant.rs

```diff
@@ -699,6 +701,42 @@ impl Tenant {
         .await
     }
 
+    pub fn create_tenant(
+        conf: &'static PageServerConf,
+        tenant_conf: TenantConfOpt,
+        tenant_id: TenantId,
+        remote_storage: Option<GenericRemoteStorage>,
+    ) -> anyhow::Result<Arc<Tenant>> {
+        let wal_redo_manager = Arc::new(PostgresRedoManager::new(conf, tenant_id));
+        create_tenant_files(conf, tenant_conf, tenant_id)?;
+        // create tenant in Active state so it is possible to issue create_timeline
+        // request. Which on timeline activation will trigger tenant activation
```

Comment:
I'm working on this.

Reply:
@hlinnaka  done, take a look here: https://github.com/neondatabase/neon/pull/2913

Reply:
I changed this before I saw your PR, so that the Tenant is created in Loading state, and we immediately call `activate()` on it to start the background jobs. What do you think of this now?

It seems a bit silly to go through the motions of loading the tenant that we just created, when we know that it doesn't contain any timelines yet. Wouldn't hurt either I guess, and it won't take long time to scan an empty directory, but still.

Reply:
Merged #2913 here